src/modules/navigation/components/MaterialRequestCenterBlade/MaterialRequestCenterBlade.tsx

```diff
@@ -100,6 +100,17 @@ const MaterialRequestCenterBlade: FC<MaterialRequestCenterBladeProps> = ({ isOpe
 		setIsEditMaterialRequestBladeOpen(true);
 	};
 
+	const onClosePersonalInfoBlade = async () => {
+		refetch();
+		const getResponse = await MaterialRequestsService.getAll({
+			isPersonal: true,
+			size: 500,
+			isPending: true,
+		});
+		dispatch(setMaterialRequestCount(getResponse.items.length));
```

Comment:
This will make 2 calls to the same endpoint. once to fetch the info for this component
and one to fill the material count in the store
Maybe its possible to set the store count using a useEffect hook?
```
useEffect(() => {
	dispatch(setMaterialRequestCount(materialRequests.items.length));
}, [materialRequests])
```

That way, we only so the network call once.